### PR TITLE
add js dom task

### PR DESCRIPTION
**Pull request by frostwolm**
### js-dom

[Demo](https://frostwolm.github.io/js-dom/) |
[Code base](https://github.com/frostwolm/js-dom)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/frostwolm/js-dom/css/app.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  box-sizing: inherit;
+}
+
+body {
+  background-color: #F1E3A7;
+  margin: 0;
+  height: 100%;
+}
+
+img{
```
Comment (A-Ostrovnyy) [inline submissions/frostwolm/js-dom/css/app.css]: Avoid styling by tag add custom styles on class.
For `<body>` and `<html>` leave but for other tags it is better to assign them a class and style them according to the class.

File: submissions/frostwolm/js-dom/css/app.css [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  list-style: none;
+  margin-left: 0;
+  padding: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.menu__item{
```
Comment (A-Ostrovnyy) [inline submissions/frostwolm/js-dom/css/app.css]: add `:hover` and `:focus` state for `.menu_item`

File: submissions/frostwolm/js-dom/css/app.css [hunk 3]
```diff
@@ -0,0 +1,115 @@
+html {
+  box-sizing: border-box;
+  height: auto;
+}
+
+*,
+*::before,
+*::after {
```
Comment (A-Ostrovnyy) [inline submissions/frostwolm/js-dom/css/app.css]: Remove unused rules.

File: submissions/frostwolm/js-dom/index.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="css/app.css">
+  <link rel="shortcut icon" href="">
+  <title>Dom project</title>
+</head>
+<body>
+  <section class="container">
```
Comment (A-Ostrovnyy) [inline submissions/frostwolm/js-dom/index.html]: Do not use the `<section>` element as a generic container; this is what `<div>` is for, especially when the sectioning is only for styling purposes.
[https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#Usage_notes](url)

File: submissions/frostwolm/js-dom/index.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    <main>
+      <article class="text">
+      </article>
+    </main>
+    <script type="text/javascript">
+      document.addEventListener(`DOMContentLoaded`, function () {
+        contentController.init(document.querySelector(`nav.menu`), document.querySelector(`article.text`));
+      });
+    </script>
+    <script src="js/app.js" type="text/javascript"></script>
```
Comment (A-Ostrovnyy) [inline submissions/frostwolm/js-dom/index.html]: Good practice: tag `<script>` should be before closing tag `<body>`.

**Comment by OleksiyRudenko** [comment]
@frostwolm The correct link to a PR fixing guide: https://gist.github.com/OleksiyRudenko/8b3ddb664308de0634b53c525e551d8b

**Comment by zonzujiro** [comment]
Looks like that PR was abandoned by the author. Therefore I will close it but we always can open it again.

**Pull request closed without merging**